src/test/java/codeu/controller/ProfileListServletTest.java

```diff
@@ -38,15 +38,15 @@ public void setup() {
 
     @Test
     public void testDoGet() throws IOException, ServletException {
-        profilePageServlet.doGet(mockRequest, mockResponse);
+        profileListServlet.doGet(mockRequest, mockResponse);
         Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
 
         User testUser = new User(UUID.randomUUID(), "Test User", "Test Password", Instant.now());
```

Comment:
Anything below this line is not actually testing anything- it never calls the `verify()` function or any of the `assert()` functions. You could modify this test to actually test that the correct user was added?